doc/classes/RichTextLabel.xml

```diff
@@ -341,14 +353,19 @@
 	</methods>
 	<members>
 		<member name="bbcode_enabled" type="bool" setter="set_use_bbcode" getter="is_using_bbcode">
+			Changes whether the label uses BBCode.
 		</member>
 		<member name="bbcode_text" type="String" setter="set_bbcode" getter="get_bbcode">
+			The BBCode text in the label.
 		</member>
 		<member name="override_selected_font_color" type="bool" setter="set_override_selected_font_color" getter="is_overriding_selected_font_color">
+			Use a custom select font color instead of the default color.
```

Comment:
Per the writing guidelines: http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-if-true-to-describe-booleans

If [code]true[/code] the label uses the custom font color. Default value: [code]false[/code].